lib/package-detail-view.coffee

```diff
@@ -201,30 +196,21 @@ class PackageDetailView extends View
     loadTime + activateTime
 
   installUpdate: ->
```

Comment:
@benogle I think we might be able to :fire: this whole method since the package card is now responsible for installing updates (and handling errors?) - what do you think?